### ISSUE TITLE
Could the display range of the horizontal screen be enlarged ?

### ISSUE THREAD

**Issue by zy9096** [appium/appium-desktop#825]
While the device using horizontal screen, its display range in appium-desktop is too small. Thus, it's hard to perform the operation on PC. Could this problem be solved in new version ?

**Comment by dpgraham** [comment]
If you enlarge the window it stretches out... is that still too small?

**Comment by zy9096** [comment]
> If you enlarge the window it stretches out... is that still too small?

Yes, still too small because of the device's screen resolution is 1920*720.
If the left area of the inspector could be adjusted, it will be fine.

**Comment by zy9096** [comment]
![20190325205422](https://user-images.githubusercontent.com/19945764/54972928-bcbbf980-4fc8-11e9-9106-9df079aa881e.png)

**Comment by AutomationSifu** [comment]
I'm experiencing the same issue with an application that has a lot of text fields and buttons.  Even with enlarging the Appium window across two monitors it's still too small.
